ode-wrapper/src/main/java/com/trihydro/odewrapper/controller/WydotTimBaseController.java

```diff
@@ -661,22 +705,31 @@ protected void expireReduceCreateSendTims(WydotTim wydotTim, TimType timType, St
         Milepost firstPoint = milepostsAll.get(0);
         Milepost secondPoint = milepostsAll.get(1);
 
-        var anchor = getAnchorPoint(firstPoint, secondPoint);
+        Milepost anchor;
+        try {
+            anchor = getAnchorPoint(firstPoint, secondPoint);
+        } catch (Utility.IdenticalPointsException e) {
+            utility.logWithDate(
+                "Identical points found during anchor point calculation, unable to generate TIM.");
+            return;
+        }

```

Comment:
Clarification (non-blocking): The new exception is handled here in the WydotTimBaseController also.